desc/grid.py

```diff
@@ -53,50 +53,69 @@ def __init__(self, nodes, sort=True):
 
         self._nodes, self._spacing = self._create_nodes(nodes)
 
-        dtheta_scale = self._enforce_symmetry()
+        self._enforce_symmetry()
         if sort:
             self._sort_nodes()
         self._find_axis()
         self._count_nodes()
-        self._scale_weights(dtheta_scale)
+        self._scale_weights()
 
     def _enforce_symmetry(self):
         """Enforce stellarator symmetry.
 
         1. Remove nodes with theta > pi.
         2. Rescale theta spacing to preserve dtheta weight.
             Need to rescale on each theta coordinate curve by a different factor.
-            dtheta should = 2pi / number of nodes remaining on that theta curve
-
-        Returns
-        -------
-        dtheta_scale : ndarray
-            The multiplicative factor to scale the theta spacing for each theta curve.
-                number of nodes / (number of nodes - number of nodes to delete)
+            dtheta should = 2pi / number of nodes remaining on that theta curve.
```

Comment:
what is meant by "theta curve"?

Reply:
the intersection of a rho and zeta surface

Reply:
Isn't that just a toroidal cross-section?

Reply:
I thought a toroidal cross-section is just a zeta surface (surface of constant zeta, which is two dimensional) and the intersection of a rho surface and zeta surface is a curve (one dimensional line).

Reply:
I see, thanks